packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -367,6 +366,16 @@ class SnackBar extends StatefulWidget {
   /// Called the first time that the snackbar is visible within a [Scaffold].
   final VoidCallback? onVisible;
 
+  /// Default vertical padding for this snack bar.
+  /// 
+  /// Defaults to 14.0.
+  /// 
+  /// This will not be overriden when setting [padding]'s top and/or bottom values.
+  /// Both paddings will just be added.
```

Comment:
Can you elaborate on this a bit more? I am not sure I understand the expected behavior from this.

Reply:
Because `SnackBar` also have a `padding` property. So, these two paddings will just add up.

Sample:
```
padding: const EdgeInsets.symmetric(vertical: 10),
singleLineVerticalPadding: 20,
```
Total vertical padding will be 30.

Reply:
Hm, this makes me wonder if we should add this. It appears to be opportune for confusion. Could we not achieve the desired effect by using the already exposed `padding`?

Reply:
@Piinks 
Yes. I was also thinking that it would be better to just use the `padding` property. I'll commit a version for this. :bow:

Reply:
@Piinks Committed: 9b459a5